config/locales/marc_records/en.yml

```diff
@@ -394,6 +394,7 @@ en:
     dedicatee: Dedicatee
     department: Department
     depositor: Depositor
+    deprecated_fields: Obsolete fields (do not use)
```

Comment:
These are from the translations files https://github.com/rism-digital/translations/blob/main/locales/en.yml

Reply:
Yes I will update the translations repo by hand when we have all the new ones

Reply:
ok